lib/components/form/default-search-form.js

```diff
@@ -13,7 +19,7 @@ export default class DefaultSearchForm extends Component {
   }
 
   static defaultProps = {
-    icons: defaultIcons,
+    icons: customIcons,
```

Comment:
Just from a variable naming perspective, it feels wrong to set a default to a variable called `customIcons`.